sqlglot/dialects/druid.py

```diff
@@ -0,0 +1,22 @@
+from sqlglot import exp, generator
+from sqlglot.dialects.dialect import Dialect
+
+
+class Druid(Dialect):
+    class Generator(generator.Generator):
+        # https://druid.apache.org/docs/latest/querying/sql-data-types/
+        TYPE_MAPPING = {
+            **generator.Generator.TYPE_MAPPING,
+            exp.DataType.Type.CHAR: "STRING",
+            exp.DataType.Type.NCHAR: "STRING",
+            exp.DataType.Type.NVARCHAR: "STRING",
+            exp.DataType.Type.TEXT: "STRING",
+            exp.DataType.Type.VARCHAR: "STRING",
+            exp.DataType.Type.UUID: "STRING",
+            exp.DataType.Type.DECIMAL: "DOUBLE",
+            exp.DataType.Type.BOOLEAN: "LONG",
+            exp.DataType.Type.TINYINT: "LONG",
+            exp.DataType.Type.SMALLINT: "LONG",
+            exp.DataType.Type.INT: "LONG",
+            exp.DataType.Type.BIGINT: "LONG",
```

Comment:
Just making sure I understand this -- are `INT`, `BOOLEAN`, `VARCHAR` etc invalid Druid SQL? Based on the docs it looks like they're supported just fine:

> The following table describes how Druid maps SQL types onto native types when running queries [...]

But Druid maps them to `LONG` etc internally. If that's the case then we don't really need this mapping, just let Druid take care of it and only add entries for the unsupported types, e.g. `NCHAR` or `UUID`, which are not mentioned in that table.

Reply:
Ah, you're right, sorry for the confusion. Let me fix this.